apps/tup-cms/src/apps/portal/templates/portal/assets.html

```diff
@@ -1,6 +1,8 @@
 {% load static %}
 
 <style>
+  @import url("{% static 'site_cms/css/build/core-styles.theme.default.css' %}") layer(base);
+
```

Comment:
mirrors theme[^1] variables from the file it overwrites https://github.com/TACC/Core-CMS/blob/task/get-core-styles-beyond-0.11.0/taccsite_cms/templates/assets_core_cms.html#L5-L6

[^1]: The themes are currently only `default` and `has-dark-logo` and only affect the header.